uf-client-ui-example/build.gradle

```
@@ -31,16 +33,25 @@ android {
         main.java.srcDirs += 'src/main/kotlin'
     }
 
-    def isRunningOnTravis = System.getenv("CI") == "true"
+    def KyneticsKeystorePass = System.getenv("KYNETICS_KEYSTORE_PASS")
+    def KyneticsKeystoreAlias = System.getenv("KYNETICS_KEYSTORE_ALIAS")
```

```suggestion
    def KyneticsKeyAlias = System.getenv("KYNETICS_KEY_ALIAS")
```

This is the alias of the key, in other words, the friendly name of the key inside the keystore.